README.md

```diff
@@ -111,9 +111,18 @@ Key | Description | Possible Values | Example
 --- | --- | --- | ---
 *name*|Unique name of route| string | teams-vul-route
 *input*|A Rego rule to match against incoming messages. If there is a match then this route will be chosen for the incoming message| Rego language statements | contains(input.message,"alpine")
+*input-files*|One or more files with Rego rules| Set of Rego language files | ["route1.rego", "route2.rego"] 
```

Comment:
'route' naming is no very good for example

Reply:
changed